### PR TITLE
Enable subset of available tests in ASDF

### PR DESCRIPTION
**Pull request by bkuehnert**
1. Tests enabled:
* case-preservation-test
* extend-test
* reference-test
* serialization-test
* symbol-import-test

2. Copied google/protobuf/descriptor.proto from https://github.com/protocolbuffers/protobuf.

### CODE REVIEW

File: tests/symbol-import-tests.lisp [hunk 1]
```diff
@@ -7,16 +7,30 @@
...
+  ;; These are here because they are exported from the automobile
+  ;; schema below and not having them causes a build error.
+  (:export #:CL-PROTOBUFS.TEST.SYMBOL-IMPORT-TEST
+	   #:SYMBOL-IMPORTER-MESSAGE-%%IS-SET
+	   #:MAKE-SYMBOL-IMPORTED-MESSAGE
+	   #:SYMBOL-IMPORTER-MESSAGE.CLEAR-IMPORTED-TYPE-FIELD
+	   #:SYMBOL-IMPORTER-MESSAGE.IMPORTED-TYPE-FIELD
+	   #:MAKE-SYMBOL-IMPORTER-MESSAGE
+	   #:SYMBOL-IMPORTED-MESSAGE-%%IS-SET
+	   #:SYMBOL-IMPORTER-MESSAGE.HAS-IMPORTED-TYPE-FIELD)
```
Comment (Slids) [inline tests/symbol-import-tests.lisp:19]: lowercase please

File: tests/cl-protobufs-tests.asd [hunk 2]
```diff
@@ -65,22 +83,25 @@
...
                  (:file "lisp-reference-tests")))
 
-   #+protobuf-file-debugged
    (:module "nested-extend-test"
     :serial t
     :pathname ""
-    :components ((:protobuf-file "extend-test.proto")
+    :components ((:protobuf-source-file "extend-base")
+		 (:protobuf-source-file "extend"
+		  :depends-on ("extend-base"))
```
Comment (Slids) [inline tests/cl-protobufs-tests.asd:102]: formatting is wrong

File: tests/symbol-import-tests.lisp [hunk 3]
```diff
@@ -7,16 +7,30 @@
 (defpackage #:cl-protobufs.test.symbol-import-test
   (:use #:cl
         #:clunit)
+  ;; These are here because they are exported from the automobile
+  ;; schema below and not having them causes a build error.
+  (:export #:CL-PROTOBUFS.TEST.SYMBOL-IMPORT-TEST
+	   #:SYMBOL-IMPORTER-MESSAGE-%%IS-SET
```
Comment (Slids) [inline tests/symbol-import-tests.lisp:13]: formatting is wrong

File: tests/symbol-import-tests.lisp (hunk 1 above)
Comment (bkuehnert) [reply tests/symbol-import-tests.lisp:19]: Uppercase is used in tests/serialization-tests.lisp, make that change as well?

File: tests/symbol-import-tests.lisp [hunk 4]
```diff
@@ -7,16 +7,30 @@
 (defpackage #:cl-protobufs.test.symbol-import-test
   (:use #:cl
         #:clunit)
+  ;; These are here because they are exported from the automobile
+  ;; schema below and not having them causes a build error.
```
Comment (Slids) [inline tests/symbol-import-tests.lisp:11]: Also, where the automobile?
(Copy past thing I'm guessing.)

**Pull request merged**